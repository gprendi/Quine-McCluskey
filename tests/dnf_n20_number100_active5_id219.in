d 20
00000000000000001000
00101000100000000000
00101000100000000100
00001000100000001000
00100000000000000000
00100000100000000000
00100000100000001100
00101000000000000000
00001000000000000100
00000000000000000100
00001000000000001100
00101000000000001000
00100000000000001000
00100000100000000100
00000000000000000000
00000000100000001000
00000000100000000100
00100000100000001000
00101000100000001100
00001000100000000100
00000000100000001100
00100000000000000100
00101000000000001100
00100000000000001100
00001000000000000000
00001000100000000000
00101000000000000100
00000000000000001100
00101000100000001000
00001000000000001000
00000000100000000000
00001000100000001100
