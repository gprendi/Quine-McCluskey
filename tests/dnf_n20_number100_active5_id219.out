p dnf 20 0
-1 -2 -4 -6 -7 -8 -10 -11 -12 -13 -14 -15 -16 -19 -20 0
